accelerator/config.h

```diff
@@ -5,24 +5,35 @@
 extern "C" {
 #endif
 
-// Address binding and port for tangle-accelerator
-#define TA_HOST "localhost"
-#define TA_PORT "8000"
-#define TA_THREAD_COUNT 10
+/**
+ * @file config.h
+ * @brief Configuration of tangle-acclerator
+ */
 
-// IRI connection setting
-#define IRI_HOST "localhost"
-#define IRI_PORT 14265
+/** @name tangle-accelerator config */
+/** @{ */
+#define TA_HOST "localhost" /**< Binding address of tangle-acclerator */
+#define TA_PORT "8000"      /**< Binding port of tangle-acclerator */
+#define TA_THREAD_COUNT 10  /**< Threads of tangle-acclerator */
```

Comment:
s/Threads/Thread count/
It is indeed a sum of the number of threads per TA instance.